enki/PhysicalEngine.h

```diff
@@ -312,6 +312,15 @@ namespace Enki
 		void setCustomHull(const Hull& hull, double mass);
 		//! Set the overall color of this object, if hull is empty or if it does not contain any texture
 		void setColor(const Color &color);
+
+		enum ButtonCode
+		{
+			LEFT = 1<<0,
```

Comment:
I would use `LEFT_MOUSE_BUTTON` as in C++98 enums are not strongly typed. So `PhysicalEngine::LEFT` is not very understandable